src/components/FlawForm.vue

```diff
@@ -102,14 +103,41 @@ const onReset = () => {
 };
 
 const displayCvssNISTForm = computed(() => {
-  const rhCvss = `${flawRhCvss.value?.score}/${flawRhCvss.value?.vector}`;
-  const nvdCvssScore = flawNvdCvssScore.toString();
-  return rhCvss !== nvdCvssScore;
+  return cvssString.value?.toString() !== flawNvdCvssScore.value?.toString();
 });
 
 const cvssString = computed(() => {
-  return `${flawRhCvss.value?.score}/${flawRhCvss.value?.vector}`;
+  const values = [flawRhCvss.value?.score, flawRhCvss.value?.vector].filter(value => !!value);
+  return values.join('/');
+});
+
+const highlightedNvdCvssScore = computed(() => {
+  let result = '';
+  const flawValue = String(flawNvdCvssScore.value);
+  const cvssValue = String(cvssString.value);
```

Comment:
If these values will always be strings, so the casting isn't necessary, unless they are null or undefined sometimes. We could adjust this in the zod schema to give them default values of `''` with `.default('')` and then we would have the benefit of knowing they would always be strings... Thoughts?

Reply:
I think the problem is that  `nistCvssScore` can be undefined and flawNvdCvssScore can be undefined. so i think we need to do like this.

`
  const nvdCvssString = computed(() => {
    const values = [nistCvssScore.value?.score, nistCvssScore.value?.vector].filter(value => !!value);
    return values.join('/') || flaw.value.nvd_cvss3 || '-';
  });
`